router/router.go

```diff
@@ -246,7 +245,8 @@ func New(cfg *config.Configuration, rateConvertor *currency.RateConverter) (r *R
 	}
 
 	syncers := usersyncers.NewSyncerMap(cfg)
-	gdprPerms := gdpr.NewPermissions(context.Background(), cfg.GDPR, adapters.GDPRAwareSyncerIDs(syncers), generalHttpClient)
+	gvlVendorIDs := bidderInfos.ToGVLVendorIDMap()
```

Comment:
Not that it really matters, but is there a reason that you took `bidderInfos.ToGVLVendorIDMap()` out into its own line?

Reply:
To make the gdprPerms line a bit shorter and to make it 100% clear that we're passing through the vendor ids.